test/vcf/assembly_checker_integration_test.cpp

```diff
@@ -39,9 +39,10 @@ namespace ebi
             std::string file_prefix = folder.parent_path().filename().string();
             std::string vcf_path = folder.string() + file_prefix + ebi::vcf::VCF_EXT;
             std::ifstream vcf_input{vcf_path};
-            std::ifstream fasta_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT};
-            std::ifstream fasta_index_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT + ebi::vcf::INDEX_EXT};
-            CHECK(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, vcf_path, fasta_input, fasta_index_input, assembly_report, outputs));
+            std::string fasta_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT};
+            std::string fasta_index_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT + ebi::vcf::INDEX_EXT};
```

Comment:
Could these be renamed as `fasta_path` and `fast_index_path`, to match the naming criteria for the VCF file? Otherwise it gets a bit confusing whether they are paths or input.